ci/setup_env.sh

```diff
@@ -61,9 +63,15 @@ which conda
 echo
 echo "update conda"
 conda config --set ssl_verify false
-conda config --set quiet true --set always_yes true --set changeps1 false
-conda install pip conda  # create conda to create a historical artifact for pip & setuptools
-conda update -n base conda
+conda config --set quiet true --set always_yes true --set changeps1 false --set channel_priority strict
+if [ "$JOB" == "pypy3" ]; then
+    conda create -n pypy3 pypy
```

Comment:
you shouldn't need to update this, what is the reason?

Reply:
In my test container, when I didn't do this, conda tried to use CPython rather than pypy3.